sections/browsers.include

```diff
@@ -4497,8 +4495,7 @@
   fully loading, the user agent must <a>update the session history with the new page</a>.
 
   User agents may add content to the <{head}> element of the {{Document}}, or
-  attributes to the element <var>host element</var>, e.g., to link to a style sheet or a
-  binding, to provide a script, to give the document a <code>title</code>, to make the media
+  attributes to the element <var>host element</var>, e.g., to link to a style sheet to provide a script, to give the document a <code>title</code>, to make the media
```

Comment:
Similar to previous comment:
` sheet to provide ` are not one concept, but two, need commas; can drop the word `to`.